registry/quay.io/biocontainers/trinity/container.yaml

```diff
@@ -5,12 +5,15 @@ description: Trinity, developed at the Broad Institute and the Hebrew University
   Jerusalem, represents a novel method for the efficient and robust de novo reconstruction
   of transcriptomes from RNA-seq data.
 latest:
-  2.13.2--ha140323_0: sha256:727ae4211b1a2fdfa33c1bb30fbf8ffc231d610c6497fe7150b031fa9a70b7b1
+  date.2011_11_26--ncurses5.9_8: sha256:eabe346a2adac1bb30aaaa62764f299638daa2a83ef6408722b0f89f8c490e20
 tags:
   2.11.0--h5ef6573_1: sha256:6d35d716aa12ba7b0c715fa8a30359b43ba9151e854dc4407b949e7b57c3a50a
   2.12.0--h5ef6573_0: sha256:4045560dad56fee946c9bcccd2ba0e373292a5adf8a2f7ba09917cbaeaa74323
   2.12.0--ha140323_1: sha256:2736617a96c30bce93a149fe5f47b4f4a290a7d21c1503619430db619d9bc3a0
   2.13.2--ha140323_0: sha256:727ae4211b1a2fdfa33c1bb30fbf8ffc231d610c6497fe7150b031fa9a70b7b1
+  date.2011_11_26--ncurses5.9_8: sha256:eabe346a2adac1bb30aaaa62764f299638daa2a83ef6408722b0f89f8c490e20
```

Comment:
This one is 4 years old. Can it be filter out ?

Reply:
Definitely.